supportedBackends/sfs/src/main/scala/cromwell/backend/impl/sfs/config/ConfigBackendLifecycleActorFactory.scala

```diff
@@ -32,6 +38,10 @@ class ConfigBackendLifecycleActorFactory(val configurationDescriptor: BackendCon
     initializationData.runtimeAttributesBuilder.definitions.toSet
   }
 
-  override lazy val fileHashingFunction: Option[FileHashingFunction] = Option(FileHashingFunction(ConfigBackendFileHashing.getMd5Result))
+  override lazy val fileHashingFunction: Option[FileHashingFunction] = {
+    logger.info(hashingStrategy.toString)
```

Comment:
The message is explicit (https://github.com/broadinstitute/cromwell/pull/1508/files#diff-1a39f56f5db2136bb3d1e898de8671a5R56)
I'll change it to debug